docs/examples/src/talk_to_a_pdf_1.py

```diff
@@ -30,9 +31,10 @@
     rag_engine=engine,
 )
 
-artifacts = PdfLoader().load(response.content)
+artifacts = PdfLoader().parse(response.content)
+chunks = TextChunker().chunk(artifacts)
 
-vector_store.upsert_text_artifacts({namespace: artifacts})
+vector_store.upsert_text_artifacts({namespace: chunks})
```

Comment:
The method name looks a little awkward now, what do you think?

Reply:
I don't think so. Artifacts are just data, chunks are a type of data. I could rename `chunks` back to `artifacts` but there might be times when you want to upload non-chunk Artifacts.